content/docs/guide/gamemodes/sticky_jump.md

```diff
@@ -10,18 +10,16 @@ tags:
-
-The version of Sticky Jump implemented in Momentum Mod has a few key changes from Team Fortress 2. Most notably, all health and ammo is gone. Instead, players are limited to having 3 stickybombs out at any given time and further restrictions are done using special triggers in the maps.
+Based on the Demoman class from Team Fortress 2, players shoot explosives that stick to surfaces and must be detonated manually, which players can then use to propel themselves around the map. Up to 3 of these arcing bombs can be used at once to go at extremely high speeds, or used in alternation to "pogo" on the floor, walls, or even in the air.
 
 ## History of Sticky Jump
 
-TODO
+Sticky Jump movement can be found in Team Fortress 2, but due to the high self-damage of stickybombs, its' prominence is limited within the main game. Most of sticky jumping is done on maps and servers specifically catered to it, inlcuding plugins that give unlimited ammo and set the players health to max on each tick, and allowed the use of some cheat-protected commands such as thirdperson.
+
+Momentum Mod's implementation of Sticky Jump has a few key changes from Team Fortress 2. Ammo/reloading is removed, and instead of having to deal with health, players are limited to having 3 stickybombs out at any given time.
 
 ## How Sticky Jump Works
 
 Typically the player shoots a stickybomb at the ground, then jumps, crouches, and detonates the stickies all in extremely quick succession. If done properly the player will go flying in a certain direction.
 
-Stickybombs take a few seconds to arm before they're able to be detonated, marked by the three circles on the HUD and a brief flash when they're ready. Because of this, pogoing (or bouncing multiple times in a row without touching the ground) can be much more involved than in rocket jumping. The player must predict where they will land and shoot there ahead of time, so that the sticky is armed and ready by the time they're above it. Essentially, the player must always place the next sticky before detonating the previous one. Also, as stickies don't require surfaces to detonate, players can sometimes detonate a sticky in the air to allow extra height, though this takes even more precision.
+Stickybombs take a fraction of a second to arm before they're able to be detonated, marked by the three circles on the HUD and a brief flash when they're ready. Because of this, pogoing (or bouncing multiple times in a row without touching the ground) can be much more involved than in rocket jumping. The player must predict where they will land and shoot there ahead of time, so that the sticky is armed and ready by the time they're above it. Essentially, the player must always place the next sticky before detonating the previous one. Also, as stickies don't require surfaces to detonate, players can detonate a sticky in the air to allow extra height, though this takes even more precision.
```

Comment:
Might just be me but I don't really like the wording at the end so I'd probably just delete everything after the comma in the last sentence `... allow extra height.` while you're updating wording in the rest of this guide